plugin-server/src/utils/db/db.ts

```diff
@@ -246,6 +246,33 @@ export class DB {
         })
     }
 
+    public async postgresBulkInsert<T extends Array<any>>(
+        // Should have {VALUES} as a placeholder
```

Comment:
very much a nit... if it's invalid to call without `{VALUES}` in the string we should error or log.

But you'd expect the tests to error if someone changed the SQL so...

Reply:
Exactly - I found that it's kind of hard to do the wrong thing here even without guards